### ISSUE TITLE
Discovery: add_document method not sending content

### ISSUE THREAD

**Issue by pthoresen** [watson-developer-cloud/python-sdk#302]
I'm running Anaconda 3.6. I used `pip` to install the latest version of the Python SDK.

I'm seeing a problem when ingesting files. Per the documentation, I'd expect this to ingest the pdf document.

```
with open('myPDF.pdf', mode='rb') as file:
    add_doc = discovery.add_document(
        <my_env>,
        <my_collection>,
        file_info=file,
        mime_type='application/pdf'
    )
```
When I do this, a new document is created in discovery, but no content is sent.

Just to make sure that I'm not going crazy, I sent the request directly using the `requests` library

```
auth = requests.auth.HTTPBasicAuth('<username>', '<password>')
files = {'file': ('myPDF.pdf', open('myPDF.pdf', mode='rb'), 'application/pdf')}
requests.post('https://gateway.watsonplatform.net/discovery/api/v1/environments/<my_environment>/collections/<my_collection>/documents?version=2017-10-16', files=files, auth=auth)
```

This worked as expected

**Comment by germanattanasio** [comment]
Can you try with the code in the `v1.0.0` We are about to release a major version that will fix this issue

**Comment by pthoresen** [comment]
Thanks, I'll try it with that version!

**Comment by ehdsouza** [comment]
We released the new version 1.0.0 on 11/13. Let us know if the issue exists

**Comment by germanattanasio** [comment]
This was fixed in v1.0.0
